delete_script.sql

```diff
@@ -0,0 +1,7 @@
+drop table appdata.vote;
```

Comment:
Great work! Could we change all the key words to capital letters? This way it is more consistent and standardized.
For example `drop table` to `DROP TABLE`.

Reply:
Updated